tools/clangformatter.py

```diff
@@ -89,6 +89,7 @@
     'src/table.c',
     'src/text.h',
     'src/toolkitAPI.c',
+    'src/toolkitAPI.h',
```

Comment:
@katmratliff, I think this like tells the auto-formatter to skip this script.

Reply:
@bemcdonnell ah - whoops, ok I'll delete that. Looks like my additions are now passing clang-format test.

Reply:
@katmratliff, ok - I don't want to delay progress due to a moot formatting point.  Please add the `src/toolkitAPI.h` back to `tools/clangformatter.py`.  :-)